scene/gui/graph_edit.h

```diff
@@ -158,6 +158,9 @@ class GraphEdit : public Control {
 	bool awaiting_scroll_offset_update;
 	List<Connection> connections;
 
+	real_t lines_thickness = 2.0f;
```

Comment:
I think this can be `float`, doesn't need to be `real_t` (`real_t` is for things that we might be to compile conditionally as `float`/`double` like in physics code).

Reply:
Fixed